extensions/qovery/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "qovery",
+  "title": "Qovery",
+  "description": "Interact with your Qovery organization",
+  "icon": "extension-icon.png",
+  "screenshots": [
+    {
+      "title": "Qovery Services List",
+      "description": "View all services in your Qovery organization",
+      "path": "metadata/service-list.png"
+    },
+    {
+      "title": "Service Actions",
+      "description": "Quick actions for each service including opening in console and viewing links",
+      "path": "metadata/service-actions.png"
+    },
+    {
+      "title": "Service Links",
+      "description": "View and manage service links",
+      "path": "metadata/service-links.png"
+    }
+  ],
+  "author": "RemiBonnet",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Find Services",
+      "description": "Interact with your Qovery organization",
+      "mode": "view"
+    }
+  ],
+  "main": "src/index.tsx",
```

Comment:
```suggestion
```
This is not a part of the schema

Reply:
Done https://github.com/raycast/extensions/pull/21255/commits/4495ec096bfc848882071a149df641c0d53d31d6